resources/views/navigation/header-button.blade.php

```diff
@@ -0,0 +1 @@
+{{-- This is shown on the first slideover of the mobile menu, this is empty by default but can be overridden --}}
```

Comment:
Not really since this the left side of the navigation header, empty is fine by default
in some projects we add a search icon for the autocomplete slideover
![image](https://github.com/user-attachments/assets/3b73a5db-c9f4-4d07-8bef-e32a2bc2f859)